docs/_includes/content.md

```diff
@@ -1213,20 +1213,151 @@ We run task **3** the output would be:
 [cargo-make] INFO - Build Done  in 0 seconds.
 ```
 
+
 <a name="usage-env"></a>
 ### Environment Variables
-cargo-make enables you to defined environment variables in several ways.<br>
-Environment variables can later be used in commands, scripts, conditions, functions and more, so it is important to have a powerful way to define them for your build.
+`cargo-make` enabled the definition of environmental variables in several ways, which can later be accessed throughout task execution.
+
+Because environmental variables play a significant role in `cargo-make`, it provides multiple declarative ways to provide them at different levels of granularity.
 
+* [Declaration](#env-declaration)
+* [Note about Ordering](#env-note-about-ordering)
 * [Global Configuration](#usage-env-config)
 * [Task](#usage-env-task)
 * [Command Line](#usage-env-cli)
 * [Env File](#usage-env-file)
 * [Global](#usage-env-global)
 
+<a name="env-declaration"></a>
+#### Declaration
+
+There are multiple ways to declare environmental variables, all of which are suited for specific suitcases.
+
+##### Simple
+
+The most ordinary one is the definition of a simple `KEY=Value` pair, which is reminiscent of tools like [dotenv](https://www.npmjs.com/package/dotenv) and [bash scripts](https://www.gnu.org/software/bash/). Values can use other variables as values, which are interpolated at runtime, using the `${variable}` syntax.
+
+> **Note:** The `$variable` syntax is currently not supported!
+
+> **Note:** Extended interpolation like `${variable:-default}` is currently unsupported!
```

Comment:
Could it be that envmnt supports non-standard interpolation through: `:default` instead?

Reply:
@indietyp you are right! it has been so long. envmnt actually supports it so i'll now add it to cargo-make.
so this can be ignored and i'll drop the comment

Reply:
I am thinking of creating a new PR to add support for the full range of interpolation as per POSIX spec: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Reply:
to envmnt? sounds good. let me check that link and we will limit it only to ${var} and not $var right

Reply:
Yep correct!